### DISCUSSION TITLE
Precompiled with Filament Change based on STOCK Ender 3 V2

### DISCUSSION THREAD

**Discussion by vaderag** [Jyers/Marlin#158] [Ideas]
I understand that there are some configurations that need changing for the load / unload filament which means that for those to be set correctly for various builds you need to compile yourself.

However, for the vast majority who are looking to simply improve their stock experience having a pre-compiled version with the parameters to suit the stock V2 would be really helpful!

**Comment by swbog** [comment]
They all work for a stock machine... right? I ran several of the 1.1.0 builds on mine just fine.

**Comment by wbvczar** [comment]
I agree with the missing changing filament option.  It was there in the 1.1 precompiled version (E3V2-ManualMesh-5x5-V4.bin) but is missing in the 1.1.1 version (I know I can recompile me own version).
